### ISSUE TITLE
[lesaloon] Exception (lesaloon): FlareSolverr was unable to process the request, please check FlareSolverr logs

### ISSUE THREAD

**Issue by dan3805** [FlareSolverr/FlareSolverr#121]
### Environment

* **FlareSolverr version**: 1.2.5
* **Last working FlareSolverr version**: 1.2.5
* **Operating system**: ubuntu
* **Are you using Docker**: yes
* **Are you using a proxy or VPN?** no
* **Are you using Captcha Solver:** no

### Description

flaresolverr is working on others website like YGG

### Logged Error Messages

2021-05-08T00:48:36.873Z INFO REQ-10 Incoming request: POST /v1
2021-05-08T00:48:36.874Z INFO REQ-10 Params: {"maxTimeout":60000,"cmd":"request.get","url":"https://lesaloonv2-0.net/index.php?page=torrents&search=&parentcategory=0&category=0&genre=0&options=0&active=0","userAgent":"Mozilla/5.0 (X11; Linux x86_64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/87.0.4280.88 Safari/537.36"}
2021-05-08T00:48:37.638Z INFO REQ-10 Cloudflare detected
2021-05-08T00:48:37.656Z INFO REQ-10 Successful response in 0.783 s
2021-05-08T00:48:37.656Z INFO REQ-10 Cloudflare Error: No challenge selectors found, unable to proceed

### Screenshots

<img width="476" alt="Capture d’écran, le 2021-05-07 à 21 09 52" src="https://user-images.githubusercontent.com/35934387/117520873-93784080-af78-11eb-8b1b-58d9670dcc22.png">

and sometime the error is only the cookies , but same error  in logs 

<img width="461" alt="Capture d’écran, le 2021-05-07 à 21 04 14" src="https://user-images.githubusercontent.com/35934387/117520746-d1289980-af77-11eb-843f-afab5a835a82.png">

**Comment by CyberPoison** [comment]
Same issue here with yggtorrent 🤔

**Comment by dan3805** [comment]
> Same issue here with yggtorrent 🤔

check your setup ,ygg is working here

**Comment by totof-l** [comment]
Hi, same issue here since today.

```
2021-05-28T15:55:51.667Z INFO REQ-0 FlareSolverr v1.2.0 listening on http://0.0.0.0:8191
2021-05-28T15:56:08.437Z INFO REQ-1 Incoming request: POST /v1
2021-05-28T15:56:08.438Z INFO REQ-1 Params: {"maxTimeout":60000,"cmd":"request.get","url":"https://www4.yggtorrent.li/engine/search?category=all&name=&description=&file=&uploader=&sub_category=&do=search&order=desc&sort=seed","userAgent":"Mozilla/5.0 (X11; Linux x86_64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/87.0.4280.88 Safari/537.36"}
2021-05-28T15:56:09.024Z INFO REQ-1 Cloudflare detected
2021-05-28T15:56:09.033Z ERROR REQ-1 No challenge selectors found, unable to proceed
2021-05-28T15:56:09.024Z INFO REQ-1 Cloudflare detected
2021-05-28T15:56:09.033Z ERROR REQ-1 No challenge selectors found, unable to proceed
2021-05-28T16:03:10.436Z INFO REQ-2 Incoming request: POST /v1
2021-05-28T16:03:10.437Z INFO REQ-2 Params: {"maxTimeout":60000,"cmd":"request.get","url":"https://www4.yggtorrent.li/","userAgent":"Mozilla/5.0 (X11; Linux x86_64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/87.0.4280.88 Safari/537.36"}
2021-05-28T16:03:10.988Z INFO REQ-2 Cloudflare detected
2021-05-28T16:03:11.001Z ERROR REQ-2 No challenge selectors found, unable to proceed
2021-05-28T16:03:11.001Z ERROR REQ-2 No challenge selectors found, unable to proceed
2021-05-28T16:04:59.630Z INFO REQ-3 Incoming request: POST /v1
2021-05-28T16:04:59.630Z INFO REQ-3 Params: {"maxTimeout":60000,"cmd":"request.get","url":"https://www4.yggtorrent.li/engine/search?category=all&name=&description=&file=&uploader=&sub_category=&do=search&order=desc&sort=seed","userAgent":"Mozilla/5.0 (X11; Linux x86_64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/87.0.4280.88 Safari/537.36"}
2021-05-28T16:05:00.119Z INFO REQ-3 Cloudflare detected
2021-05-28T16:05:00.128Z ERROR REQ-3 No challenge selectors found, unable to proceed
```

**Comment by ngosang** [comment]
It's fixed in FlareSolverr 1.2.6. It will be released today.

**Comment by ngosang** [comment]
FlareSolverr v1.2.6 released.
